integration_tests/src/main/python/window_function_test.py

```diff
@@ -272,6 +272,39 @@ def do_it(spark):
                 .withColumn('row_num', f.row_number().over(baseWindowSpec))
     assert_gpu_and_cpu_are_equal_collect(do_it, conf={'spark.rapids.sql.hasNans': 'false'})
 
+
+lead_lag_array_data_gens = [ArrayGen(int_gen, max_length=10),
+                            ArrayGen(ArrayGen(int_gen, max_length=10), max_length=10),
+                            ArrayGen(ArrayGen(ArrayGen(int_gen, max_length=10), max_length=10), max_length=10)]
+# lead and lag now support array type, so redo the tests, but just for array
+# SortExec does not support array type, so sort the result locally.
+@ignore_order(local=True)
+@approximate_float
```

Comment:
Can we remove this? There is nothing in this test that deals with floats right now, and if there were there is nothing in the test that would make it so the floats are not exact. I know this is a copy and paste from above and the one above also has it wrong.